topostats/plottingfuncs.py

```diff
@@ -75,6 +75,14 @@ def dilate_binary_image(binary_image: np.ndarray, dilation_iterations: int) -> n
     return binary_image
 
 
+def load_mplstyle(style: dict) -> None:
```

Comment:
Should the type hint here be `str`? While `plt.style.use()` can take dictionaries, they don't appear to be used in this PR for styles I think? Possibly left in from a previous iteration where you used dictionaries?

It would be odd (but technically allowed in python, as I have just found out when testing this function in a notebook) to test equality between a dictionary and a string.

If allowing dictionaries in this function is intended / wanted, then perhaps it would be good to add a check (and tests) for what type the incoming `style` is, to prevent comparison between a dictionary and a string, which could maybe cause unexpected behaviour, just a thought!

Reply:
Good spot, it should indeed be `str | Path` since in the first instance its `topostats.mplstyle` otherwise its the path to a the file that should be loaded.

Ta :+1: